oidc-conformance-tests/test_runner.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash +x
```

Comment:
add WSO2 license

Reply:
e.g. https://github.com/wso2/samples-is/blob/master/distribution/src/main/resources/get-source.sh#L3